queries/hurl/injections.scm

```diff
@@ -0,0 +1,7 @@
+; injections.scm
+(json_value) @json
+(xml) @html
+
+(multiline_string
+  (multiline_string_type) @language
+  (multiline_string_content) @content @include-children @combined)
```

Comment:
`@include-children` isn't a 0.8-compatible query. please remove it

Reply:
If you want to use `@include-children`, then please make it a 0.9-compatible version, where you use `@injection.language`, `@injection.content`, and `(#set! injection.combined)`

Reply:
It seems like it's fine to just drop include-children